package.json

```diff
@@ -29,13 +29,13 @@
     "lint": "eslint . --ext .js,.jsx",
     "prepare": "npm run clean && npm run build && husky install",
     "release": "standard-version",
-    "pretest": "cp ./test/index.test.jsx ./test/index.test.tsx && cp ./test/index.test.ssr.jsx ./test/index.test.ssr.tsx",
+    "pretest": "shx cp ./test/index.test.jsx ./test/index.test.tsx && shx cp ./test/index.test.ssr.jsx ./test/index.test.ssr.tsx",
     "test": "tsd && jest --no-cache"
   },
   "dependencies": {
     "@babel/runtime": "7.22.11",
     "dequal": "2.0.3",
-    "lru-cache": "^8.0.0"
+    "lru-cache": "^10.0.1"
   },
   "peerDependencies": {
     "axios": ">=0.24.0",
```

Comment:
you'll  need to update this too. unless we know that it's axios 1.5 and up, we should try to keep it as low as possible, e.g. 1.x

Reply:
Good spot! Thats now updated to >=1.0.0

Reply:
Thanks. Nope notifications for commits are not sent, so thanks for letting me know 👍